app/components/on-site-notifications/notification-card.js

```diff
@@ -175,6 +205,18 @@ export default Component.extend(
 			return type === notificationTypes.threadAtMention;
 		},
 
+		isArticleCommentReply(type) {
+			return type === notificationTypes.articleCommentReply;
+		},
+
+		isArticleCommentAtMention(type) {
+			return type === notificationTypes.articleCommentAtMention;
+		},
+
+		isArticleCommentReplyAtMention(type) {
```

Comment:
Do you think we can get rid of those `is` functions? It makes this file bigger and I don't think it is more readable this way. I know you just followed the pattern that was here but just wondering if we can get rid of this and just use switch statements for `iconName` and `textWithHtml`

Reply:
I suppose we could. That'll be nicer.